### PR TITLE
Merge pull request #1 from akashkw/master

### PR DESCRIPTION
**Pull request by akashkw**
Separates inventoryTest into chassisInventoryTest and  serverInventoryTest

### CODE REVIEW

File: Microservices/scripts/SMI/handlers/ChassisInventoryMicroservice.py [hunk 1]
```diff
@@ -1,37 +1,35 @@
-import json
```
Comment (michaelsteven) [inline Microservices/scripts/SMI/handlers/ChassisInventoryMicroservice.py:1]: Add license header and utf-8 declaration

File: Microservices/scripts/SMI/handlers/ChassisInventoryMicroservice.py [hunk 2]
```diff
@@ -1,37 +1,35 @@
...
-            
-            requestData = data["services"][task]["credential"]
-            url = host + data["services"][task]["url"]
-            return requestData, url
-        
-if __name__ == "__main__":    
-    test = ChassisInventoryHandler()
-    test.Inventory("summary")            
\ No newline at end of file
+import json
```
Comment (michaelsteven) [inline Microservices/scripts/SMI/handlers/ChassisInventoryMicroservice.py]: Add license header and utf-8 declaration

File: Microservices/scripts/SMI/handlers/ServerInventoryMicroservice.py [hunk 3]
```diff
@@ -1,37 +1,36 @@
-import json
```
Comment (michaelsteven) [inline Microservices/scripts/SMI/handlers/ServerInventoryMicroservice.py:1]: Add license header and utf-8 declaration

File: Microservices/scripts/SMI/tests/chassisInventoryTest.py [hunk 4]
```diff
@@ -1,46 +1,62 @@
...
-            
-            self.assertEqual(responseJson["chassisControllers"][0]["name"], value, "Controller name value doesn't match response returned from Chassis Inventory Microservice")
-            
-        except Exception as e1:
-            logger.error("ChassisInventoryMicroserviceTest: testDetails:  Exception: " + str(e1))
-            raise e1      
- 
-if __name__=="__main__":
-    unittest.main()
+import json
```
Comment (michaelsteven) [inline Microservices/scripts/SMI/tests/chassisInventoryTest.py]: Add license header and utf-8 declaration

File: Microservices/scripts/SMI/tests/serverInventoryTest.py [hunk 5]
```diff
@@ -1,211 +1,174 @@
...
-                self.assertFalse(True, "CHASSIS summary missing/incomplete in response from Inventory Microservice")
-
-        except Exception as e1:
-            logger.error("InventoryMicroserviceTest: testInventory:  Exception: " + str(e1))
-            raise e1      
-            
-            
-if __name__=="__main__":
-    unittest.main()
+import json
```
Comment (michaelsteven) [inline Microservices/scripts/SMI/tests/serverInventoryTest.py]: Add license header and utf-8 declaration

**Review by muqeethkowkab — COMMENTED** [review]
Looks Good.

**Pull request merged**